actionpack/lib/abstract_controller/helpers.rb

```diff
@@ -161,7 +161,7 @@ def modules_for_helpers(args)
             mod_name = file_name.camelize
             begin
               mod_name.constantize
-            rescue LoadError
+            rescue NameError
```

Comment:
It seems like changed original behaviors, let me figure out a better way to fix this.